solr/bin/solr

```diff
@@ -1017,7 +991,7 @@ if [[ "$SCRIPT_CMD" == "package" ]]; then
       esac
     done
   fi
-  run_package "$@"
+  run_tool package "$@"
```

Comment:
thank you!   This has been on my backlog for a while to fix...    I think you can delete `run_package` now too!